scripts/run-analyses-for-figures.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Run all analysis modules that are used for manuscript figures
+
+#enviroment settings
+set -e
+set -o pipefail
+
+# If RUN_LOCAL is used, the time-intensive steps are skipped because they cannot
+# be run on a local computer -- the idea is that setting RUN_LOCAL=1 will allow for
+# local testing running/testing of all the other figures
+RUN_LOCAL=${RUN_LOCAL:-0}
+
+# Find current directory based on this script
+WORKDIR=$(dirname "${BASH_SOURCE[0]}")
+cd "$WORKDIR"
+
+# Get base directory of project
+cd ..
+BASEDIR="$(pwd)"
+cd -
+
+analyses_dir="$BASEDIR/analyses"
+
+#################################
+
+# Run modules that cannot be run locally due to memory requirements
+if [ "$RUN_LOCAL" -lt "1" ]; then
+  # This module be run to SQL databases needed to make Figure S2 panels
+  # Note that the TCGA version would have been run in `generate-analysis-files-for-release.sh`
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-pbta.sh # Figure S2
+fi
```

Comment:
I'm confused why we would need one but not the other here?

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/2a4d691498c08cc985962447806ff5f2c2b92176/scripts/generate-analysis-files-for-release.sh#L61-L67

Is it because we're running it with the histology file in the release? (I think it's probably that, but the comment is maybe what I find confusing!)

I don't think it's a good idea to assume that someone will have the SQL database for TCGA around on their machine anyway, though.